js/src/_base/RendererPool.js

```diff
@@ -3,74 +3,84 @@ var _ = require('underscore');
 var MAX_RENDERERS = 1;
```

Comment:
Unrelated to the PR, I'm curious if you've been able to get the rendererpool to work with more than 1 max renderer?